backend/globaleaks/handlers/admin/operation.py

```diff
@@ -9,7 +9,9 @@
 from globaleaks.models.config import ConfigFactory
 from globaleaks.orm import transact
 from globaleaks.rest import errors
+from globaleaks.state import State
```

Comment:
@NCommander: state is directly accessible from handlers and jobs using self.state.

As well in case Settings would be required it would be accessible via self.state.settings

Going to apply myself a fix as part of the integration review